src/utest/test_linkage.cpp

```diff
@@ -63,8 +62,41 @@ struct crt_msg_field *linkage_test_rpc_out[] = {
 	&CMF_UINT32,
 };
 
-struct crt_req_format CRF_TEST_RPC =
-	DEFINE_CRT_REQ_FMT(linkage_test_rpc_in, linkage_test_rpc_out);
+#define TEST_LINKAGE_BASE 0x010000000
+#define TEST_LINKAGE_VER  0
+
+#define CRT_ISEQ_LINKAGE	/* input fields */		 \
+	((uint32_t)		(unused)		CRT_VAR)
+
+#define CRT_OSEQ_LINKAGE	/* output fields */		 \
+	((uint32_t)		(unused)		CRT_VAR)
+
+CRT_RPC_DECLARE(crt_linkage, CRT_ISEQ_LINKAGE, CRT_OSEQ_LINKAGE);
+CRT_RPC_DEFINE(crt_linkage, CRT_ISEQ_LINKAGE, CRT_OSEQ_LINKAGE);
+
+enum {
+	/* Client issues fetch call */
```

Comment:
not sure what these comments refer to regarding fetch/update. seems like a copy/paste from some IV code?